vital/CMakeLists.txt

```diff
@@ -258,6 +262,46 @@ endif ()
 option(KWIVER_USE_CONFIGURATION_SUBDIRECTORY
   "Look in the configuration's subdirectory for each module path (e.g. debug, release, ...)" ${default})
 
+if (WIN32)
+  set(path_sep ";")
```

Comment:
There is something subtle about the path_sep. In some of the path processing, the path_sep was interpreted by CMake and not passed undisturbed to the output. Master uses "\\073" for ";" and "\\072" for ':'. These pass through CMake and are correctly interpreted by the path parser in the plugin-loader.

Reply:
Fixed in 747140a72af0596e8e9afb5c024baad2da92d58b